1.7/products/fix_cache_is_pack.sql

```diff
@@ -0,0 +1,4 @@
+# In some cases, cache_is_pack can be wrong, this request fix it
+UPDATE ps_product p
+INNER JOIN ps_pack pa ON p.id_product = pa.id_product_pack
+SET p.cache_is_pack = 1
```

Comment:
Use generic db prefix please

Reply:
choice has been made to use `ps_` on this repository.
- this is the default so the request can be ready to use in most case
- if needed a simple seach/replace can do the job.

On the other hand, using `_DB_PREFIX_` (or another placeholder) requires a search/replace operation unless in a php script.

You can open an issue on this repo and will discuss about it :)

Reply:
Hi @SebSept 
- FYI it is not recommended to use the default `ps_` prefix, so we should promote best practices.
- it's much easier to find and replace `_DB_PREFIX_` than `ps_`
but I understand, if you have already made a decision I can't do anything I think

Reply:
Improvement is a big part of the sharing things :)  
We can do something. Nothing is fixed. Issue opened #14 

I'll mark the conservation as resolved not to meld things.